azure-local/update/update-best-practices.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Best Practices for Azure Local Update Management
+description: Learn the best practices for managing updates in Azure Local.
+author: alkohli
+ms.topic: overview
+ms.date: 08/26/2025
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.service: azure-local
+---
+
+# Best practices for managing updates in Azure Local
+
+This article provides an overview of Azure Local update management, including best practices and common pitfalls to help keep Azure Local secure, up to date, and compliant. It is intended for IT decision-makers, infrastructure architects, and operations teams responsible for Azure Local deployments.
+
+## Understand Azure Local update management
+
+Azure Local supports update management through Azure Update Manager (AUM) and Azure Resource Manager (ARM) templates.
+
+AUM provides centralized lifecycle management to apply, view, and manage updates across all Azure Local instances. ARM templates enable infrastructure-as-code for automating and scaling update configurations.
+
+Staying current with security fixes and feature improvements is important for all Azure Local components. For an overview of Azure Local updates, see [About updates for Azure Local](./about-updates-23h2.md).
+
+:::image type="content" source="./media/update-best-practices/updates.png" alt-text="Diagram that describes the key features of Azure Local update management." lightbox="./media/update-best-practices/updates.png":::
+
+## Do's for Azure Local updates
+
+Follow these practices to ensure smooth, reliable updates for Azure Local instances deployed at scale.
+
+1. **Use Azure Update Manager.**
+
+   AUM provides a centralized view to apply and manage updates across all Azure Local instances. In the Azure portal, go to **Azure Update Manager** > **Resources** > **Azure Local**.
+
+1. **Use filters to identify Azure Local resources ready for updates.**
+
+   Use filters like:
+   - Update Readiness = Healthy
+   - Status = updates available
+
+1. **Batch updates for large environments.**
```

Comment:
#2 ,#3, #4, #5 are part of #1 and not separate numbered bullets. Also, lets add a screenshot reflecting the same

Reply:
Updated #1 through #5 as bulleted list under #1. Also added a screenshot of Azure Update Manager.